### DISCUSSION TITLE
Is there any documentation to show how to integrate the python-lsp-server with vscode?

### DISCUSSION THREAD

**Discussion by 2izo** [python-lsp/python-lsp-server#190] [Configuration]

**Comment by ccordoba12** [comment]
I'm sorry to say it @2izo but there is no documentation about it and we, the maintainers of this project, don't plan to write it because we develop this server for other editors and IDEs, sorry.

However, I moved this to a discussion because other users could help you with that.

**Reply by beppo-ivel** [reply]
@ccordoba12  which IDE or editor can I try to see pylsp in action?

**Reply by ccordoba12** [reply]
Spyder, Neovim, Vim, Emacs or Jupyterlab-LSP.

**Comment by llllvvuu** [comment]
You can use my extension: https://github.com/llllvvuu/vscode-glspc . But now Microsoft has their own Python LSP which is probably better supported for VSCode.
